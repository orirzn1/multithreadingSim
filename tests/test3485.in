L14
S8
O15
N6

T0
I@0x00000000
SUBI $2, $5, 0x3
LOAD $4, $7, 0x4
LOAD $1, $2, $3
ADD $7, $1, $7
SUB $4, $7, $5
ADD $1, $7, $6
HALT $0

T1
I@0x00000000
LOAD $0, $1, $2
ADD $3, $1, $4
STORE $4, $7, $0
ADD $3, $1, $4
HALT $0

T2
I@0x00000000
SUB $1, $1, $3
ADDI $2, $5, 0x3
ADDI $0, $3, 0x3
SUB $3, $5, $4
SUBI $1, $6, 0x2
SUB $6, $4, $2
HALT $0

T3
I@0x00000000
SUBI $1, $0, 0x3
LOAD $2, $1, 0x2
SUB $4, $2, $1
ADDI $4, $0, 0x2
LOAD $0, $6, 0x4
SUB $1, $1, $3
SUBI $2, $7, 0x3
STORE $6, $4, $3
SUBI $3, $6, 0x2
HALT $0

T4
I@0x00000000
LOAD $6, $6, 0x0
LOAD $0, $4, 0x4
ADDI $1, $3, 0x4
ADD $1, $2, $2
ADD $6, $6, $5
ADD $1, $2, $1
LOAD $6, $5, $1
SUB $7, $4, $1
SUB $4, $3, $2
SUB $2, $6, $6
HALT $0

T5
I@0x00000000
SUB $7, $1, $1
ADD $7, $2, $1
ADDI $5, $7, 0x1
SUBI $1, $7, 0x3
STORE $7, $6, $5
ADDI $2, $7, 0x4
SUBI $2, $3, 0x4
ADDI $0, $7, 0x3
ADDI $0, $5, 0x1
SUBI $6, $1, 0x2
HALT $0

D@0x0
0x7
0xA
0x7
0x2
0x4
