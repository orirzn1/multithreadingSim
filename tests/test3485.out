
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFD	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x7	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x3	R1 = 0xFFFFFFFE	R2 = 0x3	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFD	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0xFFFFFFFD	R2 = 0xFFFFFFFD	R3 = 0xFFFFFFFE	R4 = 0x2	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x2	R7 = 0x0
Register file thread id 5:
	R0 = 0x2	R1 = 0xFFFFFFFD	R2 = 0xFFFFFFFC	R3 = 0x0	R4 = 0x0	R5 = 0x1	R6 = 0xFFFFFFFB	R7 = 0x0
Blocked MT CPI for this program 5.705882

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFD	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x2	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 2:
	R0 = 0x3	R1 = 0xFFFFFFFE	R2 = 0x3	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFD	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0xFFFFFFFD	R2 = 0xFFFFFFFD	R3 = 0xFFFFFFFE	R4 = 0x2	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x2	R1 = 0xFFFFFFFD	R2 = 0xFFFFFFFC	R3 = 0x0	R4 = 0x0	R5 = 0x1	R6 = 0xFFFFFFFB	R7 = 0x0
Finegrained Multithreading CPI for this program 1.235294

